packages/dht/src/dht/discovery/DiscoverySession.ts

```diff
@@ -37,33 +30,18 @@ export class DiscoverySession {
     private noProgressCounter = 0
     private ongoingClosestPeersRequests: Set<string> = new Set()
     private readonly config: DiscoverySessionConfig
-    private readonly ownPeerId: PeerID
+    private contactedPeers: Set<PeerIDKey> = new Set()
+    private joiningPath: Array<string> = [] 
```

Comment:
Do we add items to this field somewhere? Seems that it is always empty?

Reply:
removed filed